File: frontend/src/services/OrdemServicoRequest.ts
import { ItensPeca } from "../app/models/ItensPeca";
import { ItensServico } from "../app/models/ItensServico";

export interface OrdemServicoRequest {
    placaVeiculo: string;
    status: string;
    itensPeca: ItensPeca[];
    itensServico: ItensServico[];
  }

File: backend/src/main/java/com/sistemaOficina/backend/dto/OrdemServicoRequest.java
package com.sistemaOficina.backend.dto;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.util.List;

import com.sistemaOficina.backend.entity.*;

@Data
@AllArgsConstructor
public class OrdemServicoRequest {
    private String placaVeiculo;
    private Long idCliente;
    private String status;
    private List<ItensPeca> itensPeca;
    private List<ItensServico> itensServico; 
}